### ISSUE TITLE
Error installing the generator

### ISSUE THREAD

**Issue by Derdlim** [ngx-rocket/generator-ngx-rocket#198]
## I'm submitting a...

[x] Bug report  
[ ] Feature request
[ ] Documentation issue or request

## Current behavior

Error installing the generator.

Command used:
npm install -g generator-ngx-rocket

Error:
npm ERR! path C:\Users\user\AppData\Roaming\npm\node_modules\generator-ngx-rocket\cli\bin\ngx
npm ERR! code ENOENT
npm ERR! errno -4058
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod 'C:\Users\user\AppData\Roaming\npm\node_modules\generator-ngx-rocket\cli\bin\ngx'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\user\AppData\Roaming\npm-cache\_logs\2017-12-04T22_55_09_581Z-debug.log
## Expected behavior

Succesful install

## Minimal reproduction of the problem with instructions

1.- Open Node console
2.- Type npm install -g generator-ngx-rocket
3.- Error message appear

## Environment

```
- generator version: 3.1.0
- node version: 9.2
- npm version: 5.5.1
- OS:  Windows 10
```

Others:

**Comment by sinedied** [comment]
Oups, seems I missed something in the release, thanks for the report

**Comment by sinedied** [comment]
The `cli` folder was omitted from the previous release, I juste released a `3.1.1` version that fixes the issue.

**Comment by Derdlim** [comment]
Thanks for your time
